src/platform/tls_schannel.c

```diff
@@ -369,6 +374,10 @@ typedef struct _SecPkgCred_SessionTicketKeys
 #define SECPKG_ATTR_CLIENT_CERT_POLICY   0x60   // sets    SecPkgCred_ClientCertCtlPolicy
 #endif
 
+#ifndef SECPKG_ATTR_SERIALIZED_REMOTE_CERT_CONTEXT_INPROC
+#define SECPKG_ATTR_SERIALIZED_REMOTE_CERT_CONTEXT_INPROC 0x74 // returns CERT_BLOB, use only after SSPI handshake loop
+#endif
```

Comment:
This is fairly recent in Windows, right? As in, it's not supported on Win10? Assuming so, we need to document this as such.

Reply:
Yes, it's so new that it's not even in the SDK.